src/app/components/client/PlusShutdownBanner.tsx

```diff
@@ -10,53 +10,29 @@ import { CloseBtn } from "../server/Icons";
 import { useL10n } from "../../hooks/l10n";
 import { useLocalDismissal } from "../../hooks/useLocalDismissal";
 import { useHasRenderedClientSide } from "../../hooks/useHasRenderedClientSide";
-import { type ShutdownState } from "../../functions/server/getPlusShutdownState";
 
 type Props = {
-  shutdownState: ShutdownState;
   countryCode: string;
 };
 
 export const PlusShutdownBanner = (props: Props) => {
   const l10n = useL10n();
 
   const hasRenderedClientSide = useHasRenderedClientSide();
-  const localDismissal = useLocalDismissal(
-    `shutdown-banner-${props.shutdownState.hasPremium ? "plus" : "free"}-${props.shutdownState.currentMoment}`,
-  );
+  const localDismissal = useLocalDismissal(`shutdown-banner-shutdown`);
 
   if (
     !hasRenderedClientSide ||
-    // Free users who haven't run a scan have no data that would be lost:
-    (!props.shutdownState.ranScan && !props.shutdownState.hasPremium) ||
-    props.shutdownState.currentMoment === "ye-olden-days" ||
+    props.countryCode !== "us" ||
```

Comment:
I actually don't think the `countryCode` prop was being used anywhere in this component, so I'm adding it here.